src/main/webapp/WEB-INF/view/ActivityFeed.jsp

```diff
@@ -13,45 +13,58 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 --%>
+<%@ page import="java.util.List"%>
+<%@ page import="codeu.model.data.Conversation"%>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+
 <!DOCTYPE html>
 <html>
 <head>
-  <title>ActivityFeed</title>
-  <link rel="stylesheet" href="/css/main.css">
+<title>ActivityFeed</title>
+<link rel="stylesheet" href="/css/main.css">
 </head>
 <body>
 
   <nav>
-    <a id="navTitle" href="/">CodeU Chat App</a>
-    <a href="/conversations">Conversations</a>
-    <% if(request.getSession().getAttribute("user") != null){ %>
-      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-    <% } else{ %>
-      <a href="/activityfeed">Login</a>
-    <% } %>
-    <a href="/about.jsp">About</a>
+  <a id="navTitle" href="/">CodeU Chat App</a> <a href="/conversations">Conversations</a>
+  <% if (request.getSession().getAttribute("user") != null) { %>
+  <a>Hello <%=request.getSession().getAttribute("user")%>!</a>
+  <% } else { %>
+  <a href="/login">Login</a>
+  <% } %>
+  <a href="/about.jsp">About</a> <a href="/profile">Profile</a>
+  <a href="/activityfeed">Activity Feed</a>
   </nav>
 
+
   <div id="container">
-    <h1>Activity Feed</h1>
-
-    <% if(request.getAttribute("error") != null){ %>
-        <h2 style="color:red"><%= request.getAttribute("error") %></h2>
-    <% } %>
-
-    <form action="/activityfeed" method="GET">
-      <label for="username">Username: </label>
-      <br/>
-      <input type="text" name="username" id="username">
-      <br/>
-      <label for="password">Password: </label>
-      <br/>
-      <input type="password" name="password" id="password">
-      <br/><br/>
-      <button type="submit">Login</button>
-    </form>
-
-    <p>New users can register <a href="/register">here</a>.</p>
+
+	<%
```

Comment:
sorry to be so nit-picky but can you maintain the previous styling? 

<% if(request.getAttribute("error") != null){ %>

rather than <% on it's own lines. Same for throughout this file.